### DISCUSSION TITLE
在注释上新建条目笔记会弹出新窗口，但新建完之后，再次点击该注释的条目笔记会跳转到标签栏，但我希望仍然是新窗口，如何解决？

### DISCUSSION THREAD

**Discussion by Lyzzqaq** [windingwind/zotero-better-notes#1088] [Q&A]
如图，在第一条注释没有条目笔记时，点击第一条注释的小花会弹出新的条目笔记窗口（独立的）
![image](https://github.com/user-attachments/assets/548b68fe-8f4f-42d4-a345-3731e1497b7e)
![image](https://github.com/user-attachments/assets/703c20df-f79f-4888-bc2d-989e474add04)
但是，在创建了笔记之后。关闭该独立的笔记窗口，然后再次点击第一条注释的小花，会跳转到标签栏
![image](https://github.com/user-attachments/assets/76e48e7c-45d8-4701-b8b9-bd14c214a760)
我不希望跳转到标签栏因为这样很麻烦，我很多时候只是想看看这个注释写了什么，然后增添一点点内容就把它关掉。并且通常也要对照着原文去进行笔记的修改，跳转了标签栏之后，需要反复地在原文和笔记的标签之间切换，这样是不便利的。

 所以我希望它跟一开始一样仍然是新的窗口，非常希望！

@windingwind

**Comment by windingwind** [comment]
#1093
